### ISSUE TITLE
Implement ToolCommandlet for pip

### ISSUE THREAD

**Issue by hohwille** [devonfw/IDEasy#39]
Create `ToolCommandlet` for pip
Old commandlet:
https://github.com/devonfw/ide/blob/master/scripts/src/main/resources/scripts/command/pip

Duplicate of #353 

ATTENTION: Blocked by #38
